core-build/gulp-core-build-typescript/src/TypeScriptTask.ts

```diff
@@ -243,6 +266,10 @@ export class TypeScriptTask extends GulpTask<ITypeScriptTaskConfig> {
     if (!this.taskConfig.libES6Dir) {
       this.taskConfig.libES6Dir = buildConfig.libES6Folder;
     }
+
+    if (!this.taskConfig.libESNextDir) {
+      this.taskConfig.libESNextDir = buildConfig.libESNextFolder;
```

Comment:
>buildConfig.libESNextFolder; [](start = 37, length = 28)

This value isn't set in the defaults.. You should add it to the default build config:

[https://github.com/Microsoft/web-build-tools/blob/master/core-build/gulp-core-build/src/index.ts#L71](https://github.com/Microsoft/web-build-tools/blob/master/core-build/gulp-core-build/src/index.ts#L71)

We should probably have it default to "undefined", as we don't want people building these by default.

Reply:
Done. It looks like the default config is also missing `libES6Folder`. Should I add that too?